challenges/tree/README.md

```diff
@@ -40,4 +40,4 @@ Create a BinarySearchTree class
 
 - UML
 
-- ![UML](./assets/lab015.png)
\ No newline at end of file
+- ![UML](../assets/lab015.png)
```

Comment:
Code and README are looking great, would like to see that contains and add method. Once you do that and add some tests then I will regrade for you!